timescaledb/getting-started/compress-data.md

```diff
@@ -92,10 +92,10 @@ While we recommend using compression policies to compress data automatically,
 there might be situations where you need to [manually compress chunks][compress-manual]. 
 
 Use this query to manually compress chunks that consist of data older than
-2 weeks:
+2 weeks. Note that if you created the compression policy above, you will receive an error because the chunks are already compressed unless you add the `if_not_compressed` parameter shown below.
```

Comment:
```suggestion
2 weeks. If you create the compression policy above, you receive an error. This is because the chunks are already compressed unless you add the `if_not_compressed` parameter.
```